lib/pages/my_reports_pages/my_reports_page.dart

```diff
@@ -579,206 +574,204 @@ class _MyReportsPageState extends State<MyReportsPage> {
                           ),
                         ],
                       ),
-                      isMine
-                          ? Column(
-                              crossAxisAlignment: CrossAxisAlignment.start,
-                              children: <Widget>[
-                                Padding(
-                                  padding: const EdgeInsets.symmetric(
-                                      vertical: 10.0),
-                                  child: Divider(),
-                                ),
-                                Column(
-                                  children: [
-                                    Row(
-                                      children: [
-                                        Style.titleMedium(
-                                            MyLocalizations.of(
-                                                context, 'identifier_small'),
-                                            fontSize: 14),
-                                        SizedBox(
-                                          height: 4,
-                                        ),
-                                        Style.body(report.report_id,
-                                            fontSize: 14,
-                                            textAlign: TextAlign.center),
-                                      ],
-                                    ),
+                      Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: <Widget>[
+                          Padding(
+                            padding: const EdgeInsets.symmetric(
+                                vertical: 10.0),
+                            child: Divider(),
+                          ),
+                          Column(
+                            children: [
+                              Row(
+                                children: [
+                                  Style.titleMedium(
+                                      MyLocalizations.of(
+                                          context, 'identifier_small'),
+                                      fontSize: 14),
+                                  SizedBox(
+                                    height: 4,
+                                  ),
+                                  Style.body(report.report_id,
+                                      fontSize: 14,
+                                      textAlign: TextAlign.center),
+                                ],
+                              ),
+                              SizedBox(
+                                height: 12,
+                              ),
+                            ],
+                          ),
+                          report.photos != null && report.photos!.isNotEmpty
```

Comment:
Here there's a change. Diff:
```
- report.photos!.isNotEmpty
+ report.photos != null && report.photos!.isNotEmpty
```

Reply:
Is the only change done here? In the pull request the diff are being displayed wrong

Reply:
in this specific code snippet, yes